test/runner.js

```diff
@@ -0,0 +1,53 @@
+var Mocha = require('mocha')
```

Comment:
imho the runner does not belong here, you're making a meta plugin for mocha only, we'll put the test runner elsewhere